library/alloc/src/fmt.rs

```diff
@@ -354,7 +354,7 @@
 //! sign := '+' | '-'
 //! width := count
 //! precision := count | '*'
-//! type := '?' | 'x?' | 'X?' | identifier
+//! type := '?' | 'x' | 'X' | 'x?' | 'X?' | 'o' | 'b' | 'e' | 'E' | 'p'
```

Comment:
pls make sure this is in the same order as in the "Formatting traits" section

Reply:
Or even, move this change into a separate PR. I am not confident that changing the public documentation in this way is desirable and don't think there's a good reason this has to happen in the same PR.